apps/web/src/pages/Pools/IndexPoolPage/components/PoolTable.tsx

```diff
@@ -330,9 +328,8 @@ const PoolTable = ({ title, initialFilterInput }: PoolTableProps) => {
       ({ status }) => getLocalizedName(status?.label, intl),
       {
         id: "status",
-        // TO DO: Reenable when scope is added
         enableColumnFilter: false,
-        // TO DO: Reenable when relation order by added
+        // TODO: Re-enable when relation order by added.
         enableSorting: false,
```

Comment:
Is there an issue up to do this?

Reply:
Good question, you are not the first to ask 😆.

https://github.com/GCTC-NTGC/gc-digital-talent/discussions/11190#discussioncomment-10421081

Reply:
Ahaha
Yeah, TODOs without an issue or someway to search them will likely never get actioned

Reply:
[a68964b](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11339/commits/a68964be7a4350c91e971cef6a7ead7dc6a7c879)